e2e-tests/page-search.spec.ts

```diff
@@ -83,17 +83,16 @@ test('Search CJK', async ({ page, block }) => {
 })
 
 async function alias_test( block: Block, page: Page, page_name: string, search_kws: string[] ) {
+  await createRandomPage(page)
+
   const rand = randomString(10)
   let target_name = page_name + ' target ' + rand
   let alias_name = page_name + ' alias ' + rand
   let alias_test_content_1 = randomString(20)
   let alias_test_content_2 = randomString(20)
   let alias_test_content_3 = randomString(20)
 
-  // shortcut opening test
-  let parent_title = await createRandomPage(page)
-
-  await page.fill('textarea >> nth=0', '[[' + target_name + ']]')
+  await page.type('textarea >> nth=0', '[[' + target_name)
```

Comment:
I see a couple `fill`s were changed to `type`. Is the latter more useful for stability or something else ? Asking so I know when to use which api call

Reply:
Empirically, yes 🙈 It triggers the keyboard events better

Reply:
> @cnrpman Thanks for fixing tests! 👍 🚢 Unrelated to this PR, [editor.spec.ts](https://github.com/logseq/logseq/actions/runs/3586656908/jobs/6036181989) started intermittently failing on this branch (and on master). Probably worth disabling if we don't have time to fix it

I noticed the test case. Just added a delay to it.